CGPs/0020.md

```diff
@@ -0,0 +1,36 @@
+# CGP [0020]: Update randomnessBlockRetentionWindow to Extend Attestation Expiration Duration
+
+- Date: 2021-01-29
+- Author(s): @anna-carroll
+- Status: DRAFT
+- Governance Proposal ID #: [if submitted]
+- Date Executed: [if executed]
+- Discussions To: [TODO: create and add discussion link]
+
+## Overview
+
+This proposal will extend the expiration of a requested attestation from 1 HOUR -> 1 DAY, improving upon CGP 0015 which previously attempted to do so.
+
+The initial CGP 0015 attempted to extend the expiration time of attestations from 1 hour to 24 hours by changing attestationExpiryBlocks in the Attestation contract. However, the Random contract only retains randomness for a period of blocks specified by randomnessBlockRetentionWindow. Though the Attestation contract was updated to be able to query randomness between 1 and 24 hours, the Random contract remains unable to serve those requests. Therefore, the expiration time for attestations remains functionally 1 hour.
+
+See CGP 0015 for further context on the motivation for extending the expiration of requested attestations. 
```

Comment:
It's a good idea to link to CGP15, but maybe one short sentence of CGP15's motivation would make sense here as well

Reply:
totally https://github.com/celo-org/celo-proposals/pull/155/commits/8dd23e7f0efa7ab00714a3e7087d98545a01eeae